.github/workflows/java.yml

```diff
@@ -0,0 +1,23 @@
+##
+ # Java workflow to automate testing, dependency patching, and release
+ # publishing.
+ # ______________________________________________________________________________
+ #
+ # Maintained by brightSPARK Labs.
+ # www.brightsparklabs.com
+ ##
+
+name: Java Workflow
+on: [push, pull_request]
+
+jobs:
+  call-java-workflow:
+    uses: brightsparklabs/github-actions/.github/workflows/java.yml@master
+    secrets: inherit
+    # These permissions are required for Dependabot to merge PRs.
+    permissions:
+      contents: write
+      pull-requests: write
+    with:
```

Comment:
If we don't need to test on Windows, remove this `with` block. By default, tests will run on `ubuntu-latest`.